python/ray/air/examples/dreambooth/run_model.py

```diff
@@ -3,29 +3,76 @@
 
```

Comment:
Do you think renaming this file from `run_model.py` to `generate.py` will make it easier to understand?

Reply:
good idea, changed.